src/test/powersoftau_test.cpp

```diff
@@ -51,12 +51,139 @@ template<typename T> std::string to_hex(const T &v)
     return bin_to_hex(bufstream.str());
 }
 
+template<typename T> using PointSerializer = void(std::ostream &, const T &);
```

Comment:
Did you use `CamelCase` for a specific reason? Otherwise, we may want to switch to `snake_case` to be consistent